src/mjolnir/landmark_builder.cc

```diff
@@ -183,6 +194,38 @@ void LandmarkDatabase::insert_landmark(const std::string& name,
   pimpl->vacuum_analyze = true;
 }
 
+Landmark LandmarkDatabase::get_landmark(const uint32_t pkey) {
+  auto* get_landmark_stmt = pimpl->get_landmark_stmt;
+
+  sqlite3_reset(get_landmark_stmt);
+  sqlite3_clear_bindings(get_landmark_stmt);
+
+  sqlite3_bind_int(get_landmark_stmt, 1, pkey);
+  LOG_TRACE(sqlite3_expanded_sql(get_landmark_stmt));
+
+  int ret = sqlite3_step(get_landmark_stmt);
+  if (ret == SQLITE_ROW) {
```

Comment:
instead of if/else, consider doing `if (ret != SQLITE_ROW) { throw ...}`, then you don't need the else

Reply:
Ah I see there's an `else if`. Still, the comment holds, else is not necessary.